site/content/ytt/docs/latest/lang-ref-ytt-schema.md

```diff
@@ -240,8 +240,21 @@ Explicitly configures the type of the annotated node. Currently, the only suppor
 ```
 
 where:
-- `any` (`bool`) — whether or not any and all types are permitted on this node (and its children).
+- `any` (`bool`) — whether or not any and all types are permitted on this node and its children.
 
+The annotated node and its nested children are not checked by schema, and has no schema default behavior. 
+However, the annotated node and its children are simply passed-through as a data value. 
+All nested annotations are ignored.
```

Comment:
I think one can only say this safely about `@schema/...` annotations. Any templating annotation (e.g. `@yaml/text-templated-strings`) _are_ respected. This is because they are evaluated during template evaluation — _before_ the schema annotations are processed.